crates/biome_css_parser/src/syntax/at_rule/value.rs

```diff
@@ -0,0 +1,306 @@
+use biome_css_syntax::CssSyntaxKind::*;
+use biome_css_syntax::{CssSyntaxKind, TextRange, T};
+use biome_parser::diagnostic::{expect_one_of, ParseDiagnostic};
+use biome_parser::parse_lists::ParseSeparatedList;
+use biome_parser::parse_recovery::{ParseRecovery, RecoveryResult};
+use biome_parser::parsed_syntax::ParsedSyntax::Present;
+use biome_parser::prelude::ParsedSyntax::Absent;
+use biome_parser::prelude::ToDiagnostic;
+use biome_parser::{parsed_syntax::ParsedSyntax, token_set, Parser};
+
+use crate::parser::CssParser;
+use crate::syntax::parse_error::{expected_component_value, expected_identifier};
+use crate::syntax::{
+    is_at_identifier, is_at_string, is_nth_at_identifier, parse_regular_identifier, parse_string,
+};
+
+/// Checks if the current token in the parser is a `@value` at-rule.
+///
+/// This function verifies if the current token matches the `@value` rule.
+#[inline]
+pub(crate) fn is_at_value_at_rule(p: &mut CssParser) -> bool {
+    p.at(T![value])
+}
+
+/// Parses a `@value` at-rule in a CSS stylesheet.
+/// For details, see [CSS Modules Values](https://github.com/css-modules/postcss-modules-values).
+/// ```css
+/// @value my-color #f00;
+/// @value my-size 10px;
+/// @value primary, secondary from colors;
+/// @value small as bp-small, medium, large as bp-large from "./breakpoints.css";
+/// ```
+/// This function identifies and parses these `@value` rules within CSS stylesheets.
+#[inline]
+pub(crate) fn parse_value_at_rule(p: &mut CssParser) -> ParsedSyntax {
+    if !is_at_value_at_rule(p) {
+        return Absent;
+    }
+
+    if !p.options().css_modules {
+        // @value at-rule is not a standard CSS feature.
+        // Provide a hint on how to enable parsing of @value at-rules.
+        p.error(value_at_rule_not_allowed(p, p.cur_range()));
+
+        // Skip the entire rule to avoid parsing errors.
+        // Skip until the next semicolon.
+        while !p.eat(T![;]) {
+            p.bump_any();
+        }
+
+        return Absent;
+    }
+
+    let m = p.start();
+
+    p.bump(T![value]);
+
+    if is_at_value_at_rule_declaration_clause(p) {
+        parse_value_at_rule_declaration_clause(p).ok();
+    } else if is_at_value_at_rule_import_clause(p) {
+        parse_value_at_rule_import_clause(p).ok();
+    }
```

Comment:
This mean an empty `@value;` statement is allowed by the parser. Is this intended?

Reply:
Good catch!
I'll add a test case for that!
Thank you!